pkg/collector/corechecks/snmp/checkconfig/testing.go

```diff
@@ -0,0 +1,24 @@
+package checkconfig
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+// SetConfdPathAndCleanProfiles is used for testing only
+func SetConfdPathAndCleanProfiles() {
+	globalProfileConfigMap = nil // make sure from the new confd path will be reloaded
+	file, _ := filepath.Abs(filepath.Join(".", "test", "conf.d"))
+	if !pathExists(file) {
+		file, _ = filepath.Abs(filepath.Join("..", "test", "conf.d"))
+	}
+	config.Datadog.Set("confd_path", file)
+}
+
+// pathExists returns true if the given path exists
+func pathExists(path string) bool {
```

Comment:
Found [this osutil](https://github.com/DataDog/datadog-agent/blob/0454961e636342c9fbab9e561e6346ae804679a9/pkg/trace/osutil/file.go#L16-L20) from trace, don;t know if you can reuse it here

Reply:
I did saw similar pathExists elsewhere in the agent code base, but didn't wanted to depend on other packages since it's quite simple and only for testing, and not expected to change.